projects/schematics/src/migrations/mechanism/update-ssr/update-ssr-files_spec.ts

```diff
@@ -77,4 +78,34 @@ describe('Update SSR', () => {
       tree
     );
   });
+
+  describe('updateServerFile', () => {
+    it('should change nguniversal import in server.ts', async () => {
+      tree = await updateSsrSchematicRunner.runSchematic(
+        'update-ssr',
+        { name: 'schematics-test' },
+        tree
+      );
+
+      const updatedContent = tree.read('./server.ts')!.toString();
+      expect(updatedContent).toContain('@spartacus/setup/ssr');
```

Comment:
Let's use the `SSR_SETUP_IMPORT` const once it's merged to the epic branch.

Reply:
fixed